factorio_noir/category.py

```diff
@@ -15,13 +15,15 @@ class SpriteTreatment:
 
     saturation: float
     brightness: float
+    tiling: List[List[float]]
 
     @classmethod
     def from_yaml(cls, yaml_fragment: Dict[str, float]):
         """Read the sprite treatment to do from a yaml fragment."""
         return cls(
             saturation=yaml_fragment["saturation"],
             brightness=yaml_fragment["brightness"],
+            tiling=yaml_fragment.get("tiling", [[1]]),
```

Comment:
Default of `[[]]` would work better since those lines are in the file later:
```
            if tile_strength == 1:
                continue
```

Reply:
I did [[1]], cause it means a 1x1 tiling where the full strength on the transformation is applied. [], or [[]] would also work, but isn't as "pure".

I was originally wanting the number to be [0, 1] cause I wasn't sure if I needed to have different strengths within the water tiles. But Image.blend wasn't working, and it turned out that it wasn't needed for this case.

We might need more power when being compatible with other mods tho. (like the ability to specify different bri/sat for different tiles). Will cross that bridge when we get to it.

I'm not too fussed as to which way it is done.